pennylane/devices/default_qubit_autograd.py

```diff
@@ -106,8 +104,14 @@ def _asarray(array, dtype=None):
 
         return res
 
+    @staticmethod
+    def _const_mul(constant, array):
+        return constant * array
```

Comment:
Could it be that if the parent uses `qml.math.multiply`, this need not be redefined?

Reply:
Good point, but I found this needs to be redefined as `qml.math.multiply` does not handle the multiplication between a constant and array with datatype 'O' (which is used by autograd).